gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjurePlugin.java

```diff
@@ -663,12 +684,10 @@ private static <T> Set<T> mutableSetWithExtraEntry(Set<T> set, T extraItem) {
 
     static TaskProvider<WriteGitignoreTask> createWriteGitignoreTask(
```

Comment:
These write gitignore tasks are also causing task dep validation errors. Maybe not this one specifically, but there are some more internally that also have problems as they get included into source jars without the dependsOn